.github/workflows/build_and_release_for_nextcloud_with_smbclient.yaml

```
@@ -0,0 +1,47 @@
+  create:
+    name: "Creates the newest release by version"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2.3.4
+
+      # New step to read the VERSION file and set the version as an output
+      - name: Get the version
+        id: get_version
+        run: echo "nextcloud_with_smbclient_version=$(cat nextcloud-with-smbclient/VERSION)" >> $GITHUB_ENV
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:84: Double quote to prevent globbing and word splitting

(shellcheck)